client/src/api/SSOTretrieval.js

```diff
@@ -7,24 +7,48 @@
  * @description Fetch all those SSOT names and Ids, which are available for the current user
  * @param {String} userId - String including the user Id
  */
- const fetchSSOTsForUser = async (userId) => {
-    const requestString = `/ssot/getAvailableBotsForUser/${ userId }`;
-    const response = await fetch(requestString);
-    return response;
-  };
+const fetchSSOTsForUser = async (userId) => {
+  const requestString = `/ssot/getAvailableBotsForUser/${userId}`;
+  const response = await fetch(requestString);
+  return response;
+};
+
+/**
+ * @description This function renames the robot in SSOT
+ * @param {String} SSOTid - String including the SSotid
+ * @param {String} newName - String with the new RobotName
+ */
+const changeSSOTName = async (SSOTid, newName) => {
```

Comment:
Please write SSOTid in camelCase (ssotId) like all the other parameters.

Reply:
done ✅